### PR TITLE
let there be ci

### PR DESCRIPTION
**Pull request by jobez**
Time spent on this PR:

Resolves: #<Issue number>

# Pull Request type

Please check the type of change your PR introduces:

-   [ ] Bugfix
-   [ ] Feature
-   [ ] Code style update (formatting, renaming)
-   [ ] Refactoring (no functional changes, no API changes)
-   [ ] Build-related changes
-   [ ] Documentation content changes
-   [X] Testing

# What is the new behavior?

run system tests, not the EF tests themselves

# Does this introduce a breaking change?

-   [ ] Yes
-   [X] No

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+name: linters
+
+on:
+  workflow_call:
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
```
Comment (Eikix) [inline .github/workflows/linters.yml:8]:
```suggestion
    runs-on: ubuntu-latest-16-cores
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+name: test
+
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```
Comment (Eikix) [inline .github/workflows/test.yml:7]:
```suggestion
    runs-on: ubuntu-latest-16-cores
```

File: .github/workflows/linters.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        with:
+          profile: minimal
+          override: true
+      - uses: Swatinem/rust-cache@v2
+      - uses: actions-rs/cargo@v1
+        with:
+          command: check
+
+  fmt:
+    runs-on: ubuntu-latest
```
Comment (Eikix) [inline .github/workflows/linters.yml:22]:
```suggestion
    runs-on: ubuntu-latest-16-cores
```

File: .github/workflows/linters.yml (hunk 1 above)
Comment (jobez) [reply .github/workflows/linters.yml:8]: how fix https://github.com/kkrt-labs/ef-tests/actions/runs/6092630659/job/16531056251

**Review by ClementWalter — APPROVED** [review]

**Pull request merged**